### ISSUE TITLE
Issues after Windows 10 Anniversary Update

### ISSUE THREAD

**Issue by clarencephillips21** [xupefei/Locale-Emulator#268]
Windows 10 had a recent update to it called the Anniversary Update. I don't know what's new, as it appears to just be minor changes. But, after said minor changes, a few Japanese games of mine stopped working properly. It runs them incredibly slowly; far too slow to play.

I'm not sure exactly if Locale Emulator is having issues, or if something else was affected by the Anniversary Update. But, in case it is the former, is there any update for Locale-Emulator to work with Windows 10 Anniversary, or any other methods of fixing my issue?

**Comment by AmyRose1288** [comment]
well,i've been having the same problem......

after some security updates (maybe between 10586 and 14393<Anniversary Version>),all Lump of Sugar games are stopped working(only works when unicode settings has been changed to japanese),and some games randomly showing error messages (such as Access Violation or C++ Runtime error)

**Comment by AsukaLangleyfag** [comment]
I think it would be more useful for the dev if you both make a ticket properly as said in [readme](https://github.com/xupefei/Locale-Emulator/blob/master/README.md).

**Comment by Zetchzie** [comment]
This has nothing to do with Local emu afaik.  Some drivers were changed. it wouldn't happen to be an Illusion game would it?

**Comment by xupefei** [comment]
> all Lump of Sugar games are stopped working

@AmyRose1288, could you give an example? Is it happens with any of Lump of Suger games?

**Comment by AmyRose1288** [comment]
yeah, I've been testing  運命線上のφ and タユタマ２ -you're the only one-,but both of them
 Was having problems(as described above)

**Comment by AmyRose1288** [comment]
More description:
Unmei.exe was stopped due to an unknown exception, followed by a access violation error
(Same problem happens as tayutama 2)

**Comment by mireado** [comment]
um
Test with Locale emulator v2.2.1.0.pre3

it works fine.
